cron_script.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+cd $(dirname $0)
+
+. deleteENV/bin/activate
```

Comment:
Yes, it is. This is for loading the environment variables in the caller environment. If you invoke with deleteENV/bin/activate, the environment variables are defined in the called script.

This line is required for using the virtualenv.